src/explore-education-statistics-admin/src/pages/publication/components/PublicationManageTeamAccessTab.tsx

```diff
@@ -0,0 +1,73 @@
+import ReleaseContributorPermissions from '@admin/pages/publication/components/ReleaseContributorPermissions';
+import React from 'react';
+import ButtonLink from '@admin/components/ButtonLink';
+import { generatePath } from 'react-router-dom';
+import { publicationReleaseContributorsRoute } from '@admin/routes/routes';
+import { ReleaseRouteParams } from '@admin/routes/releaseRoutes';
+import releasePermissionService from '@admin/services/releasePermissionService';
+import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import { ReleaseSummary } from '@admin/services/releaseService';
+import { BasicPublicationDetails } from '@admin/services/publicationService';
+import WarningMessage from '@common/components/WarningMessage';
+
+export interface Props {
+  publication: BasicPublicationDetails;
+  release: ReleaseSummary;
+}
+
+const PublicationManageTeamAccessTab = ({ publication, release }: Props) => {
+  const {
+    value: contributors,
+    isLoading,
+    setState: setContributors,
+  } = useAsyncHandledRetry(
+    async () =>
+      releasePermissionService.listReleaseContributors(
+        publication.id,
+        release.id,
+      ),
+    [publication, release],
+  );
+
+  if (!contributors || isLoading) {
+    return <LoadingSpinner />;
+  }
+
+  const onUserRemove = async (userId: string) => {
```

Comment:
Naming convention for event handlers (not the props) is `handleX`, so this should be `handleUserRemove`

Would also suggest keeping any variable/function declarations before any if/elses. If we decide to wrap this handler in a `useCallback`, that could be a potential bug as it would break one of the React Hook rules (hooks should be deterministic).